Changelog.md

```diff
@@ -1,13 +1,8 @@
 ### Development
-[Full Changelog](http://github.com/rspec/rspec-rails/compare/v3.8.1...master)
+[Full Changelog](http://github.com/rspec/rspec-rails/compare/v3.8.2...master)
 
 Bug Fixes:
 
-* Fix issue with generator for preview specs where `Mailer` would be duplicated
-  in the name. (Kohei Sugi, #2037)
-* Fix the request spec generator to handle namespaced files. (Kohei Sugi, #2057)
-* Further truncate system test filenames to handle cases when extra words are
-  prepended. (Takumi Kaji, #2058)
 * Make the `ActiveJob` matchers fail when multiple jobs are queued for negated
```

Comment:
I am wondering if we should mention that it has been backported... 🤔

Reply:
No I think it's fine, you've mentioned its backported below

Reply:
Cool thanks